File: src/move/fibonacci.swift
func fib(_ n: Int64) -> Int64 {
    if n <= 1 {
        return n
    }
    return fib(n - 1) + fib(n - 2)
}

func main() {
    for i in 0..<50 {
        print("\(fib(Int64(i))), ", terminator: "")
    }
}

main()

File: src/move/fibonacci.go
package main

import "fmt"

func fib(n int64) int64 {
    if n <= 1 {
        return n
    }
    return fib(n-1) + fib(n-2)
}

func main() {
    for i := int64(0); i < 50; i++ {
        fmt.Printf("%d, ", fib(i))
    }
}